components/shopify_partner/sources/new_app_installs.js

```diff
@@ -0,0 +1,75 @@
+const shopify = require("../shopify_partner.app.js");
+const getAppInstalls = require("../queries/getAppInstalls");
+
+module.exports = {
+  key: "shopify_partner-new-app-installs",
+  name: "New App Installs",
+  type: "source",
+  version: "0.0.1",
+  description: "Emit new events when new shops install your app.",
+  props: {
+    shopify,
+    db: "$.service.db",
+    appId: {
```

Comment:
Does it make sense to include this as a part of the information we collect for a connected account? i.e. should a Shopify Partner account be specific to an (organization, app ID)?

Reply:
Separate question: is there a Shopify API to list app IDs? Can this prop be an async option so the user doesn't have to enter it manually?

Reply:
> Does it make sense to include this as a part of the information we collect for a connected account? i.e. should a Shopify Partner account be specific to an (organization, app ID)?

It's ad hoc, the `app.events` namespace in the GraphQL API requires a specific `app_id`, however the `transactions` namespace doesn't accept app IDs and its just transactions on the partner (or organization, same meaning).

> Separate question: is there a Shopify API to list app IDs? Can this prop be an async option so the user doesn't have to enter it manually?

I wish, I would like to add async options for a better UX, but I couldn't find an `Apps` connection in the query root. Kind of a bummer.